codexctl/__init__.py

```diff
@@ -95,24 +100,35 @@ def call_func(self, function: str, args: dict) -> None:
 
         ### Mounting functionalities
         elif function in ("extract", "mount"):
-            try:
-                from .analysis import get_update_image
-            except ImportError:
-                raise ImportError(
-                    "remarkable_update_image is required for analysis. Please install it!"
-                )
-
             if function == "extract":
                 if not args["out"]:
                     args["out"] = os.getcwd() + "/extracted"
 
                 logger.debug(f"Extracting {args['file']} to {args['out']}")
-                image, volume = get_update_image(args["file"])
+
+                try:
+                    from remarkable_update_image import UpdateImage
+                except ImportError:
+                    raise ImportError(
+                        "remarkable_update_image is required for extraction. Please install it!"
+                    ) from None
+
+                image = UpdateImage(args["file"])
                 image.seek(0)
 
                 with open(args["out"], "wb") as f:
                     f.write(image.read())
+
+                logger.info(f"Extracted image to {args['out']}")
             else:
+                try:
+                    from .analysis import get_update_image
+                    from remarkable_update_fuse import UpdateFS
+                except ImportError:
+                    raise ImportError(
+                        "remarkable_update_image and remarkable_update_fuse are required for mounting. Please install them!"
```

Comment:
```suggestion
                        "remarkable_update_fuse is required for mounting. Please install it!"
```
remarkable-update-image is a dependency, so it will be installed by installing remarkable-update-fuse.